firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -356,7 +355,11 @@ public FirebaseApp getApp() {
    * @param json The JSON format exported by the Firebase CLI.
    * @return A task that resolves once all indices are successfully configured.
    * @throws IllegalArgumentException if the JSON format is invalid
+   * @deprecated Instead of creating cache indexes manually, consider using {@link
```

Comment:
Slight grammar tweak: "Instead of creating cache indexes manually, consider using {@link
PersistentCacheIndexManager#enableIndexAutoCreation()} to let the SDK decide whether to
create cache indexes for queries running locally."

Reply:
Fix. Thank you!